datasets/wiki-text/metadata.json

```diff
@@ -0,0 +1,121 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+    "sc": "https://schema.org/",
+    "ml": "http://mlcommons.org/schema/",
+    "includes": "ml:includes",
+    "recordSet": "ml:RecordSet",
+    "field": "ml:Field",
+    "subField": "ml:SubField",
+    "dataType": "ml:dataType",
+    "source": "ml:source",
+    "data": "ml:data",
+    "applyTransform": "ml:applyTransform",
+    "format": "ml:format",
+    "regex": "ml:regex",
+    "separator": "ml:separator",
+    "references": "ml:references"
+  },
+  "@type": "sc:Dataset",
+  "@language": "en",
+  "name": "WikiText-103",
```

Comment:
Note for the reviewers:
WikiText has multiple "versions": WikiText-2 aims to be of a similar size to the PTB while WikiText-103 contains all articles extracted from Wikipedia.

I added Wikitext-103 for now. Should I also add WikiText-2? And if so, how to combine them? I could create a https://schema.org/DataCatalog instead, containing the two Datasets, but I'm afraid I'm needlessly complicating things. 
Thoughts?

Reply:
I think just having a `WikiText-103` dataset is fine. We can later on add a `WikiText-2` later on.

I am not sure we want to embark on using `sc:DataCatalog` for now, and if we do, then I'd suggest we keep the definition of such a Catalog separate from the datasets it contains (ie: have each dataset described in a separate croissant file, and the catalog in another config).

This will come up a lot. For example with COCO2014-COCO2020, which has some overlap between various years. I think it still makes sense to consider them to be different datasets and mark lineage somewhere, although I don't know how.